tsconfig.json

```diff
@@ -21,7 +21,8 @@
     "test/**/*.ts",
     "examples/**/*.ts",
     "typings/*.d.ts",
-    "node_modules/@types/**/*.d.ts"
+    "node_modules/@types/**/*.d.ts",
+    "site/static/main.ts"
```

Comment:
FYI, I'm gonna remove site from tsconfig is a subsequent PR because it's totally an overkill that site's JS file has to be inside `build` and require TS.